src/services_models/reserves.jl

```diff
@@ -63,9 +72,14 @@ function service_requirement_constraint!(
         end
     else
         for t in time_steps
+            if use_slacks
+                resource_expression = sum(reserve_variable[:, t]) + slack_vars[t]
```

Comment:
do 
`sum(reserve_variable[:, t] for t in time_periods)` the summation of variables in JuMP is more performant with the index specified.

Reply:
here we only want to sum all reserve variable for a single time period but I get your point